webapi/README.md

```diff
@@ -73,6 +73,14 @@ Running the memory creation pipeline steps in different processes. This means th
 
 6. For more information and other options, please refer to the [memorypipeline](../memorypipeline/README.md).
 
+## Adding Plugins to the WebApi
+
+You can also add plugins that will be managed by the webapi (as opposed to manged by the webapp). Soon, all plugins will be managed by the webapi.
```

Comment:
managed
```suggestion
You can also add plugins that will be managed by the webapi (as opposed to managed by the webapp). Soon, all plugins will be managed by the webapi.
```

Reply:
This section should also be consolidated with the `### Adding Custom Plugins` section below

Reply:
I have made it more explicit that the new section is for OpenAI plugins. This should make it clearer that it's different than native & semantic functions/skills/plugins.